### PR TITLE
feat: check if is at account page

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue

# Notice

# Dependency

### CODE REVIEW

File: packages/universal-header/src/components/header.js [hunk 1]
```diff
@@ -259,6 +263,16 @@ const Header = () => {
...
+  const needPrevIconAccountRoute = [
+    '/account/donation-history',
+    '/account/email-subscription',
+  ]
+  const isOnNeedPrevIconAcoountPage = _.some(needPrevIconAccountRoute, el =>
+    _.includes(pathname, el)
+  )
+  const showPrevIcon =
+    isOnArticlePage ||
+    (isOnNeedPrevIconAcoountPage && document.body.clientWidth < 768)
```
Comment (Aylie-Chou) [inline packages/universal-header/src/components/header.js:275]: Could we give it a var name or add some comment to explain the reason we need to check `clientWidth` here ?

```
// option 1 example
const mobileOnly = document.body.clientWidth < 768
const showPrevIcon = isOnArticlePage || (isOnNeedPrevIconAcoountPage && mobileOnly)

// option 2 example
(isOnNeedPrevIconAcoountPage && document.body.clientWidth < 768) // only show it on mobile

File: packages/universal-header/src/components/header.js [hunk 2]
```diff
@@ -259,6 +263,16 @@ const Header = () => {
     isHamburgerMenuOpen: showHamburger,
   }
   const isOnArticlePage = _.split(pathname, '/')[1] === 'a'
```
Comment (Aylie-Chou) [inline packages/universal-header/src/components/header.js:265]: (nice to have)
could we identify if it's on article page with [predifined path name](https://github.com/twreporter/twreporter-npm-packages/blob/master/packages/core/src/constants/entity-path.js) in core package

File: packages/universal-header/src/components/header.js (hunk 1 above)
Comment (liruchen32) [reply packages/universal-header/src/components/header.js]: fixed

File: packages/universal-header/src/components/header.js (hunk 2 above)
Comment (liruchen32) [reply packages/universal-header/src/components/header.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**